config/locales/sr.yml

```diff
@@ -695,8 +695,6 @@ sr:
 
       <p>Originally adapted from the <a href="https://github.com/discourse/discourse">Discourse privacy policy</a>.</p>
     title: Услови коришћења и политика приватности инстанце %{instance}
-  themes:
-    default: Мастодонт
```

Comment:
this was added intentionally by a human https://github.com/tootsuite/mastodon/pull/6133/files